ebu_tt_live/node/delay.py

```diff
@@ -76,6 +83,29 @@ def update_children_timing(element, timebase, delay_int):
                 update_children_timing(child.value, timebase, delay_int)
 
 
+def update_body_timing(body, timebase, delay_int):
+
```

Comment:
since this is supposed to add a body begin time maybe it should:

1. be called `insert_body_timing` or something similar
2. have an assertion at the beginning to check that there is in fact no `begin` attribute on the `body` already